src/store/flow/block.ts

```diff
@@ -17,6 +17,14 @@ export const mutations: MutationTree<IFlowsState> = {
     findBlockOnActiveFlowWith(blockId, state as unknown as IContext)
       .name = value
   },
+  block_setUserDefinedName(state, {blockId, value}) {
+    const block = findBlockOnActiveFlowWith(blockId, state as unknown as IContext)
+
+    block.ui_metadata = {
```

Comment:
Let's use the same approach to update config & vendor_metadata `by path`, eg: [block_updateConfigByPath](https://github.com/FLOIP/flow-builder/blob/master/src/store/flow/block.ts#L94).

- [ ] Create a new mutation `block_updateUiMetadataByPath` in https://github.com/FLOIP/flow-builder/blob/master/src/store/flow/block.ts, and call it here.